inst/tests/test-tags.r

```diff
@@ -570,6 +570,32 @@ test_that("Indenting can be controlled/suppressed", {
   )
 })
 
+test_that("cssList tests", {
+  expect_identical("", css())
+  expect_identical("", css())
+  expect_identical(
+    css(
+      font.family = 'Helvetica, "Segoe UI"',
```

Comment:
I'm choosing not to tackle any property-specific rules right now (I also won't wrap a URL in url() or append "px" to a number, etc.), and maybe not ever.